core/src/test/scala/org/broadinstitute/dsde/rawls/monitor/AvroUpsertMonitorSpec.scala

```diff
@@ -81,11 +81,6 @@ class AvroUpsertMonitorSpec(_system: ActorSystem)
       withApiServices(dataSource)(testCode)
     }
 
-  def withLegacyTestDataApiServices[T](testCode: TestApiService => T): T =
-    withLegacyDefaultTestDatabase { dataSource: SlickDataSource =>
-      withApiServices(dataSource)(testCode)
-    }
-
```

Comment:
most of the changes in this test are switching it to use Quicksilver; I have called out the other changes